distribution/lib/Standard/Test/0.0.0-dev/src/Main.enso

```diff
@@ -658,3 +693,67 @@ fail_match_on_unexpected_error error frames_to_skip =
     loc = Meta.get_source_location 1+frames_to_skip
     msg = "An unexpected dataflow error (" + payload.to_text + ") has been matched (at " + loc + ")."
     here.fail msg
+
+
+## PRIVATE
+   Write the JUnit XML header.
+wrap_junit_testsuites : Suite_Config -> Any -> Nothing
+wrap_junit_testsuites config ~function =
+    if config.should_output_junit then
+        config.builder.append '<?xml version="1.0" encoding="UTF-8"?>'
+        config.builder.append '<testsuites>'
+
+    result = function
+
+    if config.should_output_junit then
+        config.builder.append '</testsuites>'
+        config.output_path.parent.create_directory
+        config.output_path.write_text config.builder.toString
+
+    result
+
+
+## PRIVATE
+   Record JUnit PENDING group.
+report_pending_group : Text -> Text -> Suite_Config -> Nothing
+report_pending_group name reason config =
+    if config.should_output_junit then
+        config.builder.append ('<testsuite name="' + name + '" timestamp="' + (Time.now.format "yyyy-MM-dd'T'HH:mm:ss") + '">')
+        config.builder.append ('<testcase name="' + name + '"><skipped message="' + (reason.replace '"' '&quot;') + '" /></testcase>')
```

Comment:
Wondering if `replace` is enough or we should have some more involved mechanism for escaping these?
Even if not, I'd refactor this into a separate `escape_xml_field` (or sth like that) function. Just in case we need to modify something in the future.